src/client/pages/Offer/Introduction/DetailsModal/index.tsx

```diff
@@ -86,6 +92,57 @@ const LoadingDimmer = styled.div<{ visible: boolean }>`
   z-index: 2;
 `
 
+type FormErrors = {
+  [key: string]: string
+} & { data: Record<string, string> }
+
+function getFormErrorsFromUnderwritterLimits(
+  limits: UnderwritingLimit[],
+  invalidFieldErrorMessage: string,
+) {
+  const invalidFields = limits.reduce((acc, limit) => {
+    const { code } = limit
+    switch (code as LimitCode) {
+      case LimitCode.INVALID_BIRTHDATE:
+      case LimitCode.UNDERAGE:
+      case LimitCode.STUDENT_OVERAGE:
```

Comment:
Isn't student overage rather related to not being allowed to pick "isStudent"?

Reply:
That's a good point. Maybe it could be both.
- Changing age when isStudent is true
- Turning on the Student Flag when the age is too high

One solution could be to show an error on both isStudent flag and birthdate. What do you think?

Reply:
hmmm...the solution really is to display error messages 😅 

but that's next step. So I would say the most likely scenario is: "Changing age when isStudent is true". So we should show the error on "birth date" like now.

If you are already "too old" then I guess you can't edit "isStudent" anyways, no?

Reply:
I am not sure how this behaves. I will have to test. I can create tickets if needed

But I think I will merge this anyway as it is an improvement over what we have today.